lib/Installation/Partitioner/ExpertPartitionerPage.pm

```diff
@@ -109,4 +115,31 @@ sub press_accept_button {
     send_key('alt-a');
 }
 
+sub open_clone_partition_dialog {
+    my ($self) = @_;
+    assert_screen(EXPERT_PARTITIONER_PAGE);
+    send_key($self->{partition_table_shortcut});
+    for (1 .. 2) { wait_screen_change { send_key "down" } }
+    save_screenshot;
+    send_key "ret";
+}
+
+sub select_all_disks_to_clone {
+    my ($self, $numdisks) = @_;
+    assert_screen(CLONE_PARTITION);
+    send_key($self->{available_target_disks_shortcurt});
+    for (1 .. $numdisks) {
```

Comment:
Have you tried to use ctrl-a shortcut? It might work.

Reply:
That's probably the first thing I tried, it notworks. I don't think there is a better solution as also, an until loop creates an infinite loop here if the needle does not match.